doc/languages-frameworks/javascript.section.md

```diff
@@ -541,7 +541,7 @@ stdenv.mkDerivation (finalAttrs: {
 
   yarnOfflineCache = fetchYarnDeps {
     yarnLock = finalAttrs.src + "/yarn.lock";
-    hash = "sha256-mo8urQaWIHu33+r0Y7mL9mJ/aSe/5CihuIetTeDHEUQ=";
+    hash = "sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=";
```

Comment:
Maybe this should be different from `src.hash`?

Reply:
> Maybe this should be different from `src.hash`? or not changed.

It doesn't matter that much :)